Content.Server/_Goobstation/StationEvents/Metric/Components/FoodMetricComponent.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Content.Server.StationEvents.Events;
+using Content.Shared.FixedPoint;
+using Content.Shared.Nutrition.Components;
+using Content.Shared._EE.Silicon.Components; // DeltaV - we have a different namespace
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Generic;
+
+namespace Content.Server._Goobstation.StationEvents.Metric.Components;
+
+[RegisterComponent, Access(typeof(FoodMetricSystem))]
+public sealed partial class FoodMetricComponent : Component
+{
+    [DataField("thirstScores", customTypeSerializer: typeof(DictionarySerializer<ThirstThreshold, FixedPoint2>))]
```

Comment:
DataField tag parameters are redundant. Not sure about the customTypeSerializers
```suggestion
    [DataField(customTypeSerializer: typeof(DictionarySerializer<ThirstThreshold, FixedPoint2>))]
```